src/Sylius/Component/Channel/spec/Sylius/Component/Channel/Model/ChannelSpec.php

```diff
@@ -12,8 +12,10 @@
 namespace spec\Sylius\Component\Channel\Model;
 
 use PhpSpec\ObjectBehavior;
-use Sylius\Component\Channel\Model\ChannelInterface;
 
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
```

Comment:
IIRC some time ago there was decision to not put `@author` tags into `*Spec` files (skipping the fact that Paweł is not author of this spec).

@pjedrzejewski @aRn0D Right?